pkg/analyzer/README.md

```diff
@@ -15,9 +15,10 @@ by emailing our [list][].
 
 ## Configuring the analyzer
 
-Both `dartanalyzer` and Dart Analysis Server can be configured with either an
-`analysis_options.yaml` or `.analysis_options` file. This YAML file can control
-which files and paths are analyzed, which lints are applied, and more.
+Both `dartanalyzer` and Dart Analysis Server can be configured with an
+`analysis_options.yaml` file (Using an `.analysis_options` file is deprecated).
```

Comment:
I think "Using" shouldn't be capitalized.  Or a period put after "file" to complete the sentence.